README.md

```diff
@@ -21,7 +21,7 @@ We've applied the philosophy of _"do one thing and do it well."_ We are focused
 * Nano utilizes a novel [block-lattice](https://github.com/nanocurrency/nano-node/wiki/Block-lattice) architecture, unlike conventional blockchains used in many other cryptocurrencies.
 * The network requires minimal resources, no high-power mining hardware, and can process high transaction throughput.
 * Offers instantaneous transactions with zero fees and unlimited scalability, making Nano an ideal solution for peer-to-peer transactions.
-* As of August 2018, the Nano network has processed over twelve million blocks with an unpruned ledger size of only 4.6GB.
+* As of March 2019, the Nano network has processed over 21 million blocks with an unpruned ledger size of only 10GB.
```

Comment:
I thought to put: `[unpruned](https://developers.nano.org/roadmap/ledger-pruning/)`. What do you think?